tidy3d/components/simulation.py

```diff
@@ -5814,3 +5814,106 @@ def from_scene(cls, scene: Scene, **kwargs) -> Simulation:
         )
 
     _boundaries_for_zero_dims = validate_boundaries_for_zero_dims()
+
+    def _make_pec_frame(self, obj: Union[ModeSource, InternalAbsorber]) -> Structure:
```

Comment:
ah, all these functions: `_make_pec_frame`, `_modal_plane_frames`, `_finalized`, and `_validate_finalized` were moved to `AbstractYeeGridSimulation`, you need to move any changes from here to there